client/src/hooks/datafiles/mutations/useTrash.ts

```diff
@@ -0,0 +1,107 @@
+import { useSelector, useDispatch, shallowEqual } from 'react-redux';
+import { useMutation } from '@tanstack/react-query';
+import { apiClient } from 'utils/apiClient';
+import Cookies from 'js-cookie';
+
+export async function trashUtil({
+  // Per TypeScript, declare variables...
```

Comment:
You can remove these comments

Reply:
Yeah, some of those comments were more "notes to self" than anything else.
Those specific comments have now been removed.